vnext/Microsoft.ReactNative/XamlUIService.h

```diff
@@ -27,6 +25,9 @@ struct XamlUIService : XamlUIServiceT<XamlUIService> {
   static void SetXamlRoot(IReactPropertyBag const &properties, xaml::XamlRoot const &xamlRoot) noexcept;
   static xaml::XamlRoot GetXamlRoot(IReactPropertyBag const &properties) noexcept;
 
+  static void SetIslandWindow(IReactPropertyBag const &properties, uint64_t hwnd);
```

Comment:
>; [](start = 81, length = 1)

Should they be noexcept? Generally noexcept method saves about 1Kb of binary space versus a method without it.

Reply:
yes good idea